lib/erb_lint/corrector.rb

```diff
@@ -17,11 +17,22 @@ def corrections
     end
 
     def corrector
-      RuboCop::Cop::Corrector.new(@processed_source.source_buffer, corrections)
+      BASE.new(@processed_source.source_buffer, corrections)
     end
 
-    def diagnostics
-      corrector.diagnostics
+    if ::RuboCop::Version::String.to_f >= 0.87
```

Comment:
Looking at the tests, it seems that you need to 

```suggestion
    if ::RuboCop::Version::STRING.to_f >= 0.87
```